tests/MongoDB.Driver.Core.TestHelpers/JsonDrivenTests/CommandStartedEventAsserter.cs

```diff
@@ -62,23 +69,43 @@ private void AdaptExpectedUpdateModels(List<BsonDocument> actualModels, List<Bso
                 var actualModel = actualModels[i];
                 var expectedModel = expectedModels[i];
 
-                if (expectedModel.Contains("multi") && expectedModel["multi"] == false && !actualModel.Contains("multi"))
+                if (!actualModel.Contains("multi"))
```

Comment:
Probably this change is equal to the previous one. Except for one additional case which appeared in current tests.

Reply:
In particular this test: https://github.com/DmitryLukyanov/mongo-csharp-driver/blob/046972bbd4ee548920d682d11e9c948026777b27/tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/tests/bulk.json has `multi=false` and `upsert=false` in the actualModel, but doesn't have these values in the expected

Reply:
Removed previous version